### PR TITLE
注文のバグを修正しました

### PR DESCRIPTION
**Pull request by SekiMoemi**
確認お願いします。

### CODE REVIEW

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 1]
```diff
@@ -11,6 +11,7 @@
 import jp.co.example.ecommerce_a.domain.Order;
 import jp.co.example.ecommerce_a.domain.User;
 import jp.co.example.ecommerce_a.service.ShowOrderConfirmService;
+import jp.co.example.ecommerce_a.service.ShowShoppingCartService;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:14]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Wrong lexicographical order for 'jp.co.example.ecommerce_a.service.ShowShoppingCartService' import. Should be before 'org.springframework.web.bind.annotation.RequestMapping'.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 2]
```diff
@@ -24,6 +25,9 @@ public class ShowOrderConfirmController {
 
 	@Autowired
 	private ShowOrderConfirmService showOrderConfirmService;
+
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:28]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 3]
```diff
@@ -24,6 +25,9 @@ public class ShowOrderConfirmController {
 
 	@Autowired
 	private ShowOrderConfirmService showOrderConfirmService;
+	
+	@Autowired
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:29]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:29]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'member def modifier' has incorrect indentation level 8, expected level should be 2.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 4]
```diff
@@ -24,6 +25,9 @@ public class ShowOrderConfirmController {
...
+	private ShowShoppingCartService showShoppingCartService;
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:30]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java [hunk 5]
```diff
@@ -37,7 +41,7 @@ public String showOrderConfirm(String orderId, @AuthenticationPrincipal LoginUse
 		User loginUser = showOrderConfirmService.findUserById(userId);
 		model.addAttribute("user", loginUser);
 
-		Order order = showOrderConfirmService.showOrderConfirm(Integer.parseInt(orderId));
+		Order order = showShoppingCartService.showShoppingCart(userId);
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/ShowOrderConfirmController.java:44]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java [hunk 6]
```diff
@@ -37,7 +37,7 @@ public class OrderController {
 	 */
 	@PostMapping("")
 	public String order(@Validated OrderForm form, BindingResult result) throws ParseException {
-		
+		System.out.println(form.toString());
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java:40]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java [hunk 7]
```diff
@@ -49,11 +49,11 @@ public String order(@Validated OrderForm form, BindingResult result) throws Pars
 			result.addError(fieldError);
 		}
 		
-		if (result.hasErrors()) {
-			return "order_confirm";
-		}
+//		if (result.hasErrors()) {
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java:52]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java [hunk 8]
```diff
@@ -49,11 +49,11 @@ public String order(@Validated OrderForm form, BindingResult result) throws Pars
...
+//			return "redirect:/showOrderConfirm";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java:53]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java [hunk 9]
```diff
@@ -49,11 +49,11 @@ public String order(@Validated OrderForm form, BindingResult result) throws Pars
...
+//		}
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java:54]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.CommentsIndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Comment has incorrect indentation level 0, expected is 2, indentation should be the same level as line 56.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java:54]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

File: src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java [hunk 10]
```diff
@@ -49,11 +49,11 @@ public String order(@Validated OrderForm form, BindingResult result) throws Pars
...
 		
 		orderService.order(form);
-		return "redirect:/order_finished";
+		return "redirect:/showOrderConfirm/finishedOrder";
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java:57]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/controller/OrderController.java:57]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

File: src/main/java/jp/co/example/ecommerce_a/service/OrderService.java [hunk 11]
```diff
@@ -52,7 +52,7 @@ public void order(OrderForm form) throws ParseException {
 		SimpleMailMessage mailMessage = new SimpleMailMessage();
 		mailMessage.setSubject("注文が完了しました");
 		mailMessage.setText("注文が完了しました。ありがとうございます！");
-		mailMessage.setTo(targetOrder.getUser().getEmail());
+		mailMessage.setTo(targetOrder.getDestinationEmail());
```
Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:55]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.FileTabCharacterCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Line contains a tab character.

Comment (SekiMoemi) [inline src/main/java/jp/co/example/ecommerce_a/service/OrderService.java:55]: ⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.indentation.IndentationCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'method def' child has incorrect indentation level 16, expected level should be 4.

**Review by yamada-kensyu — APPROVED** [review]
承認しました！ありがとうございます

**Pull request merged**